### PR TITLE
Update in ds_helper to support import to Pinnacle

### PR DESCRIPTION
**Pull request by CaB15**
I have updated the function add_refd_frame_of_ref_sequence in ds_helper to support the import of files into the treatment planning system Pinnacle. There was a note in the code that the previous version should be checked. In Pinnacle, the selection of and random UID for Frame of Reference was not accepted during the import.

### CODE REVIEW

File: rt_utils/ds_helper.py [hunk 1]
```diff
@@ -98,16 +98,22 @@ def add_patient_information(ds: FileDataset, series_data):
 
 def add_refd_frame_of_ref_sequence(ds: FileDataset, series_data):
     refd_frame_of_ref = Dataset()
+    try:
+        # Updated to support import of files into the treatment planning system Pinnacle
+        FrameOfRef=series_data[0]['FrameOfReferenceUID'].value
+    except:
+        FrameOfRef = generate_uid()
```
Comment (awtkns) [inline rt_utils/ds_helper.py:105]:
```suggestion
        refd_frame_of_ref.FrameOfReferenceUID = series_data[0]['FrameOfReferenceUID'].value
    except:
        refd_frame_of_ref.FrameOfReferenceUID = generate_uid()
```

File: rt_utils/ds_helper.py [hunk 2]
```diff
@@ -98,16 +98,22 @@ def add_patient_information(ds: FileDataset, series_data):
...
     refd_frame_of_ref.FrameOfReferenceUID = (
```
Comment (awtkns) [inline rt_utils/ds_helper.py:106]:
```suggestion
```

File: rt_utils/ds_helper.py [hunk 3]
```diff
@@ -98,16 +98,22 @@ def add_patient_information(ds: FileDataset, series_data):
...
-        generate_uid()
-    )  # TODO Find out if random generation is ok
+        FrameOfRef
+    )
```
Comment (awtkns) [inline rt_utils/ds_helper.py:108]:
```suggestion
```

**Comment by awtkns** [comment]
Thanks for this @CaB15! By chance would you be able to add a test for this?

 @asim-shrestha I think this looks good to me, thoughts?

**Comment by awtkns** [comment]
Relates to #29

**Comment by CaB15** [comment]
Dear Adam,

Sorry, this is the first time I have tried to support a git project. Thus I am not quite sure what you mean by a test. I guess it is something that you should be able to run in the future to test that the code is still producing the expected result. May I ask you to provide some information on examples of tests and how to do that? If so, I might be able to add a test for the support of the project

Best,
Carsten


Carsten Brink, Ph.D.
Professor og Hospitalsfysiker / Professor and Medical Physicist
Klinisk Institut og Radiofysisk laboratorium / Institute of Clinical Research and Laboratory of Radiation Physics
Syddansk Universitet og Odense Universitetshospital / University of Southern Denmark and Odense University Hospital
DK-5000 Odense C
Denmark
(Phone (+45) 29 79 64 79 – SMS not possible)
e-mail: ***@***.******@***.***>

**Comment by asim-shrestha** [comment]
Hi @CaB15, my apologies for the late response. This looks good! In terms of testing examples, you can view the tests folder to get started. We use a framework called Pytest. I can also tackle the tests myself this wednesday

**Comment by asim-shrestha** [comment]
Updated in #65 

Closing

**Pull request closed without merging**